packages/api-plugin-fulfillment-method-shipping-flat-rate/src/mutations/createFlatRateFulfillmentRestriction.js

```diff
@@ -0,0 +1,35 @@
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import ReactionError from "@reactioncommerce/reaction-error";
+import restrictionSchema from "../util/restrictionSchema.js";
+
+const inputSchema = new SimpleSchema({
+  restriction: restrictionSchema,
+  shopId: String
+});
+
+/**
+ * @method createFlatRateFulfillmentRestrictionMutation
```

Comment:
If you are going to use this declaration it needs to match the filename I think

Reply:
Removed 'Mutation' from func declaration as per another review comment